vtune/README.md

```diff
@@ -0,0 +1,81 @@
+## VTune analysis
+
+First, the code has been run under VTune on DevCloud "cluster" and we obtained the following picture:  
+
+![](./img/vtune/old/new_core.c_ua1.png) 
+
+As we can see, VTune marks the most CPU consuming task as "Outside any known module", and lots of lines are grayed out and marked "unreliable" when pointed with a mouse. It looks like we do not have proper access to hardware counters on Intel machine, so we went to a local laptop to see more detailed results. Thus the following picture has been obtained:
+
+![](./img/vtune/original_bottomup_uarc.png)
+
+Now we do not have any "Outside any known module" and can see that the function "evolve interior" does, in fact, consume the most resources. Here are the few summaries that VTune gave us:
+
+![](img/vtune/original_memory_summary.png)
+
+We see that the bandwidth we get is rather high, around 28 GB/sec.
+
+![](img/vtune/original_summary_the.png)
+
+The CPU utilization seems low, but that is because the software detects all available CPUs while we couldn't run the code on 6 cores due to how the distribution of the data is implemented. So we were only using 4 cores and got effective utilization of 3.623.
+
+
+
+Now lets look at our memory usage:
```

Comment:
These images are too tough at first sight. Please, crop them and point out the details we want to focus.